primitives/io/src/lib.rs

```diff
@@ -1142,6 +1142,147 @@ pub trait Crypto {
 	}
 }
 
+/// Interfaces for working with elliptic curves related types from within the runtime.
+#[runtime_interface]
+pub trait EllipticCurves {
```

Comment:
This should also be moved to `primitives/artwork` crate. No need to have this here directly in `sp-io`

Reply:
I moved it to `primitives/crypto-ec-utils` crate.